ledger/acctupdates.go

```diff
@@ -1895,7 +1910,10 @@ func (au *accountUpdates) vacuumDatabase(ctx context.Context) (err error) {
 		}
 	}()
 
+	start := time.Now()
+	ledger_vacuum_count.Inc(nil)
 	vacuumStats, err := au.dbs.wdb.Vacuum(ctx)
+	counterMicros(ledger_vacuum_micros, start)
```

Comment:
In this particular case, I think that we should use the existing `vacuumElapsedTime` and add it to the `ledger_vacuum_micros` counter. I'd rather avoid having several "variations" for the same operation.